src/docker/docker.cpp

```diff
@@ -103,6 +103,7 @@ Try<Docker*> Docker::create(const string& path, bool validate)
     return docker;
   }
 
+/*
```

Comment:
Remove the assumption of cgroups in the Docker containerizer.

Mesos on Triton uses our [elastic Docker service](https://docs.joyent.com/public-cloud/api-access/docker) that treats the entire data center as a single Docker host, and doesn't run tasks in containers in virtual machines.